src/app/components/tabview/tabview.css

```diff
@@ -6,6 +6,10 @@
     overflow: hidden;
 }
 
+.p-tabview-scrollable > .p-tabview-nav {
```

Comment:
why did you add this new css to tabview.css ?

Reply:
You are right there are overlayed changes.     From memory.  tabview.css changes were because there is a wrap layout definition that is not done properly.  there is a class to wrap the tabs however it is broken and does not actually wrap tabs when all the rest of the CSS works towards this.    This is from memory and it was over 12 months ago.

Reply:
Ok, reverted this changes.